### PR TITLE
Added Pattern Creation Game

### PR DESCRIPTION
**Pull request by ananyag309**
Issue #4694 
Added Pattern Creation Game
@ishita-43 please have a look, i have followed all the contributing rules. Have a check and add gssoc and level label.
thank you

https://github.com/kunjgit/GameZone/assets/145869907/7bf392dc-101e-4e15-a3db-73be00b6e9c2

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @ananyag309 ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

**Comment by ananyag309** [comment]
@ishita-43 ma'am please check I have made necessary changes.

**Comment by ananyag309** [comment]
@ishita-43 @kunjgit sorry for disturbing again. it's been a week. please have a check & please letme know if any further changes need to be made.

File: README.md [hunk 1]
```diff
@@ -825,7 +825,7 @@ This repository also provides one such platforms where contributers come over an
 |[Wheel_of_Fortunes](https://github.com/Saipradyumnagoud/GameZone/tree/main/Games/Wheel_of_Fortunes)|
 |[Tic-tac-toe](https://github.com/Saipradyumnagoud/GameZone/tree/main/Games/Tic-tac-toe)|
 |[Quest_For_Riches](https://github.com/kunjgit/GameZone/tree/main/Games/Quest_For_Riches)|
-
+|[Pattern Creation Game](https://github.com/ananyag309/GameZone_ggsoc/tree/main/Games/Pattern_Creation_Game)|
```
Comment (ishita-43) [inline README.md:828]: change your name with the repository name as other did eg kunjgit/GameZone

**Review by ishita-43 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ananyag309) [reply README.md]: done

**Review by ishita-43 — APPROVED** [review]
good to go!!

**Comment by kunjgit** [comment]
Thank you  @ananyag309 , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**